backend/src/main/java/cz/mzk/fofola/configuration/AsyncConfiguration.java

```diff
@@ -1,9 +1,25 @@
 package cz.mzk.fofola.configuration;
 
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.scheduling.annotation.EnableAsync;
+import org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor;
+
+import java.util.concurrent.Executor;
 
 @Configuration
 @EnableAsync
 public class AsyncConfiguration {
+
+    @Bean
+    public Executor taskExecutor() {
```

Comment:
Tim vytvaris globalni taskExecutor pro celou appku, a to na 2 vlakna, coz urcite pro celou appku neni vhodny. Potrebujes tenhle taskExecutor jen pro generovani PDF, dej beanu nazev a ten specifikuj v @Async PDF generatoru, vic https://www.baeldung.com/spring-async#1-override-the-executor-at-the-method-level